src/States/MapViewStateDraw.cpp

```diff
@@ -221,30 +221,38 @@ void MapViewState::drawResourceInfo()
 void MapViewState::drawRobotInfo()
 {
 	// CC hasn't been placed yet.
-	if (ccLocationX() == 0) { return; }
+	if (ccLocation() == NAS2D::Point<int>{0, 0}) { return; }
```

Comment:
Just a thought. Why is the `ccLocation` defaulting to a valid tile? Why not default it to `{-1,-1}`?

Reply:
Yeah, I kind of wondered that too. We should set a constant somewhere to represent that invalid position.

Probably a separate issue though. There are likely going to be a few places in the code where we need to make that change.